client/settings/payment-methods-list/payment-method.tsx

```diff
@@ -51,12 +54,20 @@ interface PaymentMethodProps {
 const PaymentMethodLabel = ( {
 	id,
 	label,
+	accountFees,
 }: {
 	id: string;
 	label: string;
+	accountFees?: Record< string, FeeStructure >;
 } ): React.ReactElement => {
 	const { chip, chipType = 'warning' } = usePaymentMethodAvailability( id );
 
+	const discountFee =
+		accountFees?.[ id ]?.discount && accountFees[ id ].discount.length > 0
+			? accountFees[ id ].discount[ 0 ]
+			: null;
+	const hasDiscount = discountFee && discountFee.discount;
```

Comment:
Nitpick: I think this could be shortened a little to: 


```suggestion
  const discountFee = accountFees?.[ id ]?.discount?.[ 0 ];
  const hasDiscount = discountFee?.discount;
```